src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,84 @@
+import { BsArrowRight } from "react-icons/bs";
+import { Link, useNavigate } from "react-router-dom";
+import "./Login.css";
+import { getUser } from "../../services";
+import { useUser } from "../../context";
+export const Login = () => {
+  const { setUser } = useUser();
+  const navigator = useNavigate();
+
+  const formHandler = (e) => {
+    e.preventDefault();
+    const [email, password] = e.target.elements;
+
+    (async () => {
+      try {
+        const { data, status } = await getUser(email.value, password.value);
```

Comment:
You don't need to `await` it again as `getUser ` will give resolved promise.

Reply:
Please correct me if i'm wrong but `getUser` is an `async` function so they should return a promise and to capture its returned data we have to use `await`.